source/_posts/2024-06-11-roadmap-2024h1.markdown

```diff
@@ -0,0 +1,162 @@
+
+Secondly, of the three Open Home values, privacy, choice, and sustainability, while we have a pretty solid understanding of choice and some understanding of sustainability, the idea of privacy can benefit from more clarity. We want a better understanding of what privacy really means to a home and its residents.
+
+#### Technology research (R&D)
+
+Yes, large language models (LLMs) are a hot trend, but that’s not why we’re exploring them. We know there are very interesting problems they can solve in the home, and with the amount of data our users collect in their history, we believe that there is much power underneath Home Assistant yet to be unearthed and it is uniquely positioned to be the smart home platform for AI. [Our AI vision](/blog/2024/06/07/ai-agents-for-the-smart-home/) lays down the foundation for what we want to do with AI and LLMs next.
+
+In order to help AI make better decisions, we need a better understanding of the context of what each device actually is. To do so, we need to refine our information architecture. For instance, we recently rolled out floors helping Home Assistant better understand your house. This is no small task but could be vital in implementing better voice or even AI in the home.
+
+* * *
+
+## Home Assistant Cloud: “Peace of Mind”
+
+[Home Assistant Cloud](https://www.nabucasa.com/), powered by our commercial partner Nabu Casa brings, to the many, the hard-to-reach features of Home Assistant usually reserved for power users with expensive hardware at home, with no compromise on privacy.
+
+Currently, Home Assistant Cloud provides the simplest and safest way to access your Home Assistant instance anywhere in the world and connect to third-party cloud-connected voice assistants, like Google’s and Alexa’s offerings. Our newest ability added to Cloud is the ability to offload the resource-intensive tasks of speech-to-text and text-to-speech. This allows our users to have power-efficient hardware at home, like the Home Assistant Green, which uses three watts and performs tasks usually reserved for hundred-watt home servers.
+
+For the next phase of Home Assistant Cloud, we want to continue to build features that adhere to our mission as stated above. The theme for the upcoming initiatives revolve around providing peace of mind for our users:
+
+#### Current priority: Hardening security
+
+We want to harden our security, providing more control for remote access and Two-Factor Authentication (2FA) for your Nabu Casa account.
+
+#### Next priority: Encrypted cloud backup
+
+This would allow users to have a more resilient installation of Home Assistant, so if your instance completely breaks down it can be restored from the cloud.
```

Comment:
```suggestion
This would allow users to have a more resilient installation of Home Assistant, so if your instance completely breaks down, it can be restored from the cloud.
```